tests/behat/theme_boost_union_block_region.feature

```diff
@@ -0,0 +1,553 @@
+@theme @theme_boost_union  @theme_boost_union_block_config @javascript
+Feature: Theme boost union block config
+  In order to add more functionality to pages
+  I need to add blocks to pages
+
+  Background:
+    Given the following "users" exist:
+      | username | firstname | lastname | email |
+      | student1 | Student | 1 | student1@example.com |
+      | student2 | Student | 2 | student2@example.com |
+      | teacher  | teacher | name | teacher@example.com |
+    And the following "courses" exist:
+      | fullname | shortname | format |
+      | Course 1 | C1 | topics |
+    And the following "activities" exist:
+      | activity   | name                   | intro                         | course | idnumber    | section |
+      | assign     | Test assignment name   | Test assignment description   | C1     | assign1     | 0       |
+      | book       | Test book name         | Test book description         | C1     | book1       | 0       |
+      | chat       | Test chat name         | Test chat description         | C1     | chat1       | 1       |
+    And the following "course enrolments" exist:
+      | user | course | role |
+      | student1 | C1 | student |
+      | student2 | C1 | student |
+      | teacher  | C1  | teacher |
+    And I log in as "admin"
+    When I turn editing mode on
+
+  Scenario: Add a block to a left region.
```

Comment:
It would be worthwhile trying to compress all scenarios into a Scenario outline (see https://github.com/moodle-an-hochschulen/moodle-theme_boost_union/blob/master/tests/behat/theme_boost_union_looksettings_branding.feature#L87-L103 for an example) to reduce the length of this feature file and to improve maintainability.